jwst/regtest/test_nirspec_fs_spec3.py

```diff
@@ -34,13 +34,12 @@ def test_nirspec_fs_spec3(run_pipeline, rtdata_module, fitsdiff_default_kwargs,
 
     output = f"jw93045-o010_s00003_nirspec_f290lp-g395h-subs400a1_{suffix}.fits"
     rtdata.output = output
-
-    # Get the truth files
     rtdata.get_truth(f"truth/test_nirspec_fs_spec3/{output}")
 
-    # Set looser tolerance for resampled s2d comparison
+    # Adjust tolerance for machine precision with float32 drizzle code
     if suffix == "s2d":
-        fitsdiff_default_kwargs["atol"] = 1e-5
+        fitsdiff_default_kwargs["rtol"] = 1e-2
```

Comment:
This is up to 1%, right? That's huge. I don't like the idea of the routine tests running day after day on jwcalibdev being able to hide that big of a difference. I think I would be more in favor of having a smaller tolerance and just have folks realize that it's normal to see differences between their system and jwcalibdev.

Reply:
It is.  It's only a few pixels (7 in this case) that have that high of a tolerance.  But if the test is supposed to be effective at allowing development and catching bugs, this tolerance will catch bugs, but will allow development.

Reply:
And I'm pretty sure the affected pixels are edge effects in this case were NIRSpec FS has `NaN` mapping for some of the pixels in the bounding box for the curved spectral trace, i.e. pixels that do not come through the optics from the sky.  So this is sort-of-a thresholding effect on pixels of the trace that are either in-or-out.

Reply:
Good question!  I would prefer to actually fix `drizzle` to use `float64` internally.  Maybe you know somebody who is good at C programming?!  😉

Reply:
It is strange that there is a loss of about 4-5 digits. This does not seem "right". Why is this happening?